javascript/javascript1/week2/Student-manager.js

```diff
@@ -0,0 +1,40 @@
+const class07Students = [];
+addStudentToClass('Rajesh');
+addStudentToClass('Soujanya');
+addStudentToClass('Aashrith');
+addStudentToClass('');
+addStudentToClass('Anand');
+addStudentToClass('Rajesh');
+addStudentToClass('Student5');
+addStudentToClass('Student6');
+addStudentToClass('Student7');
+addStudentToClass('Queen Margrethe');
+getNumberOfStudents();
+console.log(class07Students);
+
+function addStudentToClass(studentName) {
+  if (studentName === 'Queen Margrethe'){
```

Comment:
what happened if you add queen twice?

Reply:
yes got it

Reply:
Thankyou so much for your review..... :)